bloggies_backend/src/routes/bookmarks.ts

```diff
@@ -1,42 +1,42 @@
 import { NextFunction, Request, Response } from "express";
 import { ensureLoggedIn } from "../middleware/auth";
 import express from "express";
-import Favorite from "../models/favorite";
+import Favorite from "../models/bookmarks";
```

Comment:
Tiny thing here, but the file is "bookmark.ts" (singular). Got a TS error! (Or change file name to plural)

Also on this line, change Favorite to Bookmark